src/common/hdfs/HdfsCommandHelper.cpp

```diff
@@ -13,30 +13,81 @@ namespace hdfs {
 StatusOr<std::string> HdfsCommandHelper::ls(const std::string& hdfsHost,
                                             int32_t hdfsPort,
                                             const std::string& hdfsPath) {
-    auto command = folly::stringPrintf("hdfs dfs -ls hdfs://%s:%d%s",
+    auto command = folly::stringPrintf("hadoop fs -ls hdfs://%s:%d%s",
                                        hdfsHost.c_str(), hdfsPort, hdfsPath.c_str());
-    LOG(INFO) << "Running HDFS Command: " << command;
-    auto result = ProcessUtils::runCommand(command.c_str());
-    if (result.ok()) {
-        return result.value();
+    LOG(INFO) << "Start Running HDFS Command: [" << command << "]";
+    auto resultStatus = ProcessUtils::runCommand(command.c_str());
+    if (!resultStatus.ok()) {
+        LOG(INFO) << "HDFS Command Failed." << resultStatus.status().toString();
+        return resultStatus.status();
+    }
+
+    auto result = std::move(resultStatus).value();
+
+    if (folly::StringPiece(result).startsWith("ERR")) {
+        LOG(INFO) << "HDFS Command Failed. " << result;
+        return Status::Error(result);
     } else {
-        return Status::Error(folly::stringPrintf("Failed to run %s", command.c_str()));
+        LOG(INFO) << "HDFS Command Finished. [" << command << "], output: [" << result << "]";
+        return result;
     }
 }
 
 StatusOr<std::string> HdfsCommandHelper::copyToLocal(const std::string& hdfsHost,
                                                      int32_t hdfsPort,
                                                      const std::string& hdfsPath,
                                                      const std::string& localPath) {
-    auto command = folly::stringPrintf("hdfs dfs -copyToLocal hdfs://%s:%d%s %s",
```

Comment:
there are any different between `hdfs dfs` and `hadoop fs` ?

Reply:
The later is the recommended usage in the newer version.